4-projects/modules/infra_pipelines/versions.tf

```diff
@@ -19,11 +19,13 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.50"
+      // remove "<= 4.30.0" after new provider release with https://github.com/hashicorp/terraform-provider-google/pull/12248
```

Comment:
Rather than totally remove, you might want to exclude the problematic versions, for example `!= 4.30`.

Reply:
suggestion accepted :)